TRENZ.Lib.RazorMail.Core/Interfaces/IMailClient.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+using TRENZ.Lib.RazorMail.Models;
+
+namespace TRENZ.Lib.RazorMail.Interfaces;
+
+public interface IMailClient
+{
+    /// <summary>
+    /// The default "From" address.
```

Comment:
maybe add explanations what the Default* properties are useful for (and that you can override them per-message)

Reply:
Done